fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBJEJournal.java

```diff
@@ -394,4 +407,141 @@ public BDBEnvironment getBdbEnvironment() {
         return bdbEnvironment;
     }
 
+    /**
+     * start batch write
+     * for BDB: start transaction.
+     */
+    @Override
+    public void batchWriteBegin() throws InterruptedException, JournalException {
+        if (currentTrasaction != null) {
+            throw new JournalException(String.format(
+                    "failed to begin batch write because has running txn = %s", currentTrasaction));
+        }
+
+        JournalException exception = null;
+        for (int i = 0; i < RETRY_TIME; i++) {
+            try {
+                // sleep before retry
+                if (i != 0) {
+                    Thread.sleep(SLEEP_INTERVAL_SEC * 1000);
```

Comment:
Will the sleep will be too long?

Reply:
Since I am lack of online experience, I use the same value as in BDBJEJournal here.